scalac-scoverage-plugin/src/main/scala/scoverage/report/ReportWriter.scala

```diff
@@ -0,0 +1,118 @@
+package scoverage.report
+
+import java.io.File
+
+import org.rogach.scallop.ArgType.V
+import org.rogach.scallop.{singleArgConverter, ValueConverter, Scallop}
+import scoverage.{IOUtils, Serializer, Constants, Coverage}
+
+import language.implicitConversions
+
+object ReportWriter {
+
+  private def dir(base: File, name: String) = {
+    val directory = new File(base, name)
+    directory.mkdirs()
+    directory
+  }
+
+  def writeReports(outputDirectory: File,
+                   baseDirectory: File,
+                   coverage: Coverage,
+                   coverageOutputCobertura: Boolean,
+                   coverageOutputXML: Boolean,
+                   coverageOutputHTML: Boolean,
+                   coverageDebug: Boolean): Unit = {
+
+    if (coverageOutputCobertura) {
+      new CoberturaXmlWriter(baseDirectory, dir(outputDirectory, "cobertura")).write(coverage)
+    }
+
+    if (coverageOutputXML) {
+      val xmlDir = dir(outputDirectory, "xml")
+      new ScoverageXmlWriter(baseDirectory, xmlDir, false).write(coverage)
+      if (coverageDebug) {
+        new ScoverageXmlWriter(baseDirectory, xmlDir, true).write(coverage)
+      }
+    }
+
+    if (coverageOutputHTML) {
+      new ScoverageHtmlWriter(baseDirectory, dir(outputDirectory, "html")).write(coverage)
+    }
+
+  }
+}
+
+object Reports extends App {
```

Comment:
It seems from their documentation that `--help` produces a usage summary. I'll add some meaningful messages to make that more helpful.